jobs/keepalived/templates/bin/utils/openstack/generate_token

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
just a question , how stable are these Openstack API's ? Should we have retries for them ?

Reply:
Till now haven't faced any failure. But definitely can introduce retry. Good point!